pkg/jobs/hive/hive.go

```diff
@@ -757,21 +759,21 @@ func waitForMCV(client clientv1.Client, clusterName string, clusterNamespace str
 		}, mcv); err != nil {
 			if i == 5 {
 				klog.Warning(err)
-				return errMsg
+				return err
```

Comment:
You will need to double check this.
Do you want to return the `err` from the client call or the `err` from param?
Same comment for the below.

Reply:
Yep good catch Mike. I changed the error variable name from the client. In the if statement that's checking the for client error we return the error from the client. For other places we return the error from the param. Hope this makes sense.